CHANGELOG.md

```diff
@@ -1,3 +1,53 @@
+## 0.9.0-alpha+1
+
+> NOTE: This code is considered production quality, but depends on angular:
+> 5.0.0-alpha+1. The alpha tag represents the evolving nature of the AngularDart
+> api, not code quality (5.0.0-alpha+1 is used in production Google apps).
+
+**NOTE**: As of `angular 5.0.0-alpha+1` [`dependency_overrides`][dep_overrides]
+are **required**:
+
+```yaml
+dependency_overrides:
+  analyzer: ^0.31.0-alpha.1
+```
+
+This is because Angular is starting to use and support the Dart 2.0.0 SDK, which
+is evolving. We expect to no longer require overrides once we are at a beta
```

Comment:
I'm not sure if I agree with this change in wording. We don't have any overrides in the angular_components package but anyone that uses it will be required to have a dependency override in their pubspec.yaml to build.

Reply:
I'm ok if you prefer sticking to the current wording.